Kinvey-Xamarin/Auth/User.cs

```diff
@@ -1072,6 +1138,30 @@ private EmailVerificationRequest buildEmailVerificationRequest(string userID)
 			return email;
 		}
 
+		private RealtimeRegisterRequest BuildRealtimeRegisterRequest(string userID, string deviceID)
+		{
+			var urlParameters = new Dictionary<string, string>();
+			urlParameters.Add("appKey", ((KinveyClientRequestInitializer)client.RequestInitializer).AppKey);
+			urlParameters.Add("userID", userID);
+
+			var realtimeRegister = new RealtimeRegisterRequest(client, userID, deviceID, urlParameters);
```

Comment:
Agreed, will remove the constructor arg, since it is only required as a URL parameter.